xml/System.IO.Pipelines/FlushResult.xml

```diff
@@ -87,9 +86,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>
-          <see langword="true" /> if the <see cref="T:System.IO.Pipelines.PipeWriter" /> is complete; otherwise, <see langword="false" />.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation was completed.</summary>
+        <value><see langword="true" /> if the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation was completed; otherwise, <see langword="false" />.</value>
```

Comment:
(and again)

```suggestion
        <value><see langword="true" /> if the reader is no longer reading data written to the <see cref="T:System.IO.Pipelines.PipeWriter" />; otherwise, <see langword="false" />.</value>
```